x-pack/plugins/security_solution/server/lib/detection_engine/signals/bulk_create_ml_signals.ts

```diff
@@ -81,10 +82,10 @@ const transformAnomalyResultsToEcs = (results: AnomalyResults): SearchResponse<E
 };
 
 export const bulkCreateMlSignals = async (
-  params: BulkCreateMlSignalsParams
+  params: BulkCreateMlSignalsParams,
+  buildRuleMessage: BuildRuleMessage
```

Comment:
nit: `params` already includes things like `logger` and `services`, why is `buildRuleMessage` a separate argument here?

Reply:
Gooood catch I'll swap it out.